include/deal.II/grid/grid_generator.h

```diff
@@ -2147,6 +2247,9 @@ namespace GridGenerator
    * @note Since @p input and @p output have different spatial dimensions, no
    * manifold objects are copied by this function regardless of the value of
    * @p copy_manifold_ids.
+   *
+   * @note This function is available through the python interface as
+   * `input.extrude_triangulations(n_slices, height, result)`
```

Comment:
```suggestion
   * `input.extrude_triangulation(n_slices, height, result)`.
```
It is `extrude_triangulation` in c++.